Code/GraphMol/MolStandardize/testTautomer.cpp

```diff
@@ -715,9 +715,9 @@ void testCanonicalize() {
   TautomerEnumerator te(new TautomerCatalog(tautparams.get()));
 
   for (const auto &itm : canonTautomerData) {
-    std::unique_ptr<ROMol> mol{SmilesToMol(itm.first)};
+    ROMOL_SPTR mol{SmilesToMol(itm.first)};
```

Comment:
I don't understand the reason for the changes from unique_ptr to ROMOL_SPTR that you made in this file. The pointers aren't being shared or used elsewhere so I don't think there's any reason to make them shared. Did I miss something?

Reply:
No reason other than `ROMOL_SPTR` reads nicer and there is no `ROMOL_UPTR` `typedef`, and the fact that in the context of that test both would work. But you are right, no serious reason, so I reverted them. I am also going to write 100 times "You shall not make useless and confusing changes to existing tests that waste reviewers' time".